src/utils/contracts.ts

```diff
@@ -128,6 +140,8 @@ export class Contracts {
         usdc: getToken(networkId, 'usdc').address,
         usdcYVault: getToken(networkId, 'yvUsdc').address,
         yRegistry: getContractAddress(networkId, 'yRegistry'),
+        apeVaultWrapper: (deploymentInfo as any)[networkId].apeVaultWrapper
+          .address,
```

Comment:
We don't deploy apeVaultWrapper but use apeVaultFactory to create apeVaultWrapper. So, we will be initialising this on runtime dynamically.